examples/hybrid_quantum_algorithms/VQE_Chemistry/VQE_chemistry_braket.ipynb

```diff
@@ -352,7 +352,7 @@
     "from openfermion import MolecularData\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.    ansatz_ckt_zero = ansatz_ckt(0)
</pre><p>Why not just print the circuit with the unbound free parameter? Then follow up with a little blurb about binding the parameter, and <em>then</em> run the circuit. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Fixed